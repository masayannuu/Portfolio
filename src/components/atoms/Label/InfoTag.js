import React from 'react'
import styled from 'styled-components'
import { Tag } from 'reactbulma'

const StyledInfo = styled(Tag)`
  font-feature-settings: 'pkna';
  letter-spacing: normal;
`

const InfoTag = props => <StyledInfo className="is-rounded">{props.children}</StyledInfo>

export default InfoTag
