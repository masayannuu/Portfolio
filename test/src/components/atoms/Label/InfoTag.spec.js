import React from 'react'
import Enzyme from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import InfoTag from '../../../../../src/components/atoms/Label/InfoTag'

describe('Label InfoTag Atoms', () => {
  it('should render InfoTag', () => {
    const wrapper = Enzyme.shallow(<InfoTag>InfoTag</InfoTag>)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
