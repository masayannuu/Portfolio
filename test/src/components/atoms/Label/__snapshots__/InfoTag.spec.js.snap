// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label InfoTag Atoms should render InfoTag 1`] = `
<InfoTag__StyledInfo
  className="is-rounded"
>
  InfoTag
</InfoTag__StyledInfo>
`;
